plusdeck/dbus/client.py

```diff
@@ -162,8 +165,9 @@ def main(
     echo.mode = output
 
     async def load() -> None:
-        client = DbusClient()
-        ctx.obj = Obj(client=client, log_level=log_level, output=output)
+        bus: SdBus = sd_bus_open_user() if user else sd_bus_open_system()
```

Comment:
I figured out that the default bus is the system bus when started with a non-user systemd unit: https://www.freedesktop.org/software/systemd/man/latest/sd_bus_default.html based on me being able to get past a "where is the bus tho" error, I seem to understand this correctly.